entc/integration/packagealias/ent/generate.go

```diff
@@ -0,0 +1,7 @@
+// Copyright 2019-present Facebook Inc. All rights reserved.
```

Comment:
I prefer to keep this in `integration/ent`. There's already a similar use case, but for GoType (not enums): https://github.com/ent/ent/blob/master/entc/integration/ent/schema/task.go#L26-L27